lib/internal/net.js

```diff
@@ -68,11 +68,23 @@ function makeSyncWrite(fd) {
   };
 }
 
+function isLoopback(host) {
+  const hostLower = host.toLowerCase();
+
+  return (
+    hostLower === 'localhost' ||
+    hostLower.startsWith('127.') ||
+    hostLower.startsWith('[::1]') ||
+    hostLower.startsWith('[0:0:0:0:0:0:0:1]')
+  );
+}
```

Comment:
Can you add a source for this information?

What about 0.0.0.0 addresses?

Reply:
Thanks for the info! IMO you should add a comment with the IANA links for reference